projects/packages/forms/src/dashboard/inbox/style.scss

```diff
@@ -506,3 +506,11 @@ body.jetpack_page_jetpack-forms-admin {
 	gap: 16px;
 	align-items: center;
 }
+
+.jp-forms__inbox__unread-indicator {
+	color: #d63638;
+	font-size: 8px;
+	margin-right: 4.5px;
+	margin-left: -12px;
+	vertical-align: middle;
+}
```

Comment:
The dot color is red here and in testing. It's also quite small. In the PR description, it's blue and a bit larger, which I like more. That may just be taste, and obviously not blocker. 

<img width="1497" height="509" alt="unread-status" src="https://github.com/user-attachments/assets/9b790dff-71ae-451a-92e8-e22452c1f656" />